src/main/riddl/Members/members.riddl

```diff
@@ -5,6 +5,7 @@ type MemberMap is mapping from MemberId to MembersContext.Info
 
 context MembersContext  is {
     //Member Details
+    type StatusUpdates is any of {Active, Inactive, Suspended} briefly "Internal status of an Organization."
```

Comment:
This isn't well-named.  The confusing part for me is how an enumerator has anything to do with "Update". This is a representation of a state machine's state.  These aren't internal either, there ought to be public API to change and retrieve these states.  

How about "MemberState" ?

Reply:
In all places I've touched there is a command to change state. In the other 2 BCs (Org & Tenant) the status is included in MetaInfo and can be retrieved from there.

I can change all Status types according to `[Entity]State` pattern, add Status to MetaInfo of Member if desired, and add explicit commands for retrieving states if desired.

The only quibble I have with those is that I want to keep the word `Status` here to differentiate Active enum from `record Active[Entity]State` that may contain one of these enums. Unless we want to eliminate the name `Active` from those `State` records... I've considered just calling them `[Entity]State` to eliminate possible confusion of having 2 `Active` keywords in the same place

Reply:
re:
> I can change all Status types according to [Entity]State pattern, add Status to MetaInfo of Member if desired, and add explicit commands for retrieving states if desired.

Sounds about right. Let's not bother adding the explicit commands for now until there is a demonstrated reason to do so. We don't have a use case for it so lets wait until there is one. 

> The only quibble I have with those is that I want to keep the word Status here to differentiate Active enum from record Active[Entity]State that may contain one of these enums ...

So, there is quite a conundrum here because of the dual nature of RIDDL's state machine support.  I would suggest you think about RIDDL's "state" keyword at the meta-level. In the application that comes out from it, that meta-level disappears.  The "Status" you refer to are the definitional states of a state machine and so I prefer the word "state" for this. If you look at the definitions of these words, state can be attributed to inanimate or animate things. But status is mostly about people, law, society or a situation. The definition of "State" include:
* A condition of being in a stage or form, as of structure, growth, or development.
* The condition of a physical system with regard to phase, form, composition, or structure.
That fits a state machine pretty well.  "Status" on the other hand doesn't have an equivalent. All of its connotations are these:
* Position relative to that of others; standing.
* High standing; prestige.
* The legal character or condition of a person or thing.
* The state of affairs; the situation: synonym: state.
* Standing or position as regards rank or condition.
* Position of affairs.
* In law, the standing of a person before the law in the class of persons indicated by his or her legal qualities; the relation fixed by law in which a person stands toward others or the state.
* A person’s [condition](https://www.wordnik.com/words/condition), [position](https://www.wordnik.com/words/position) or [standing](https://www.wordnik.com/words/standing) relative to that of others.
The thing that I am concerned about with Status is its relative nature as in "standing", "position relative to", "relative to that of others".  State doesn't impact any of this relativeness to me. 

Consider the above and then decide for yourself, we don't need to "quibble" further.

Reply:
The keyword may disappear, but the records themselves have State in the name. However I do prefer dropping the dual application of `Active` in naming conventions. I just went through some possible word choices and I think if we call them `Established[Entity]State` instead of `Active[Entity]State` it will work well because we usually use `Establish[Entity]` command to change to enumerated state. In Event I can use `ScheduledEventState`, and things like that where appropriate. 

I will change the `UpdateStatuses` to `[Entity]State` as well.